browser_action/index.html

```diff
@@ -95,6 +95,15 @@
 
       <div class="panel-section-separator"></div>
 
+      <div class="panel-formElements-item">
+        <div>
+          <input type="date" name="dateMaxSearch" id="dateMax" />
+          <label for="dateMax">Date maximale de recherche (MM/JJ/YYYY)</label>
```

Comment:
Ici l'intégration du i18n manque

```suggestion
          <label for="dateMax" data-i18n="dateMaxLabel>Date maximale de recherche (MM/JJ/YYYY)</label>
```

`dateMaxLabel` doit ensuite être défini dans `_locales/*/messages.json`.

Je suggère comme traduction allemande:

```
"maxDateLabel": {
    "message": "Termin finden bis zum (DD. MM. YYYY)",
    "description": "Bis zu diesem Datum wird nach einem Termin gesucht"
  },
```

Reply:
Rajouté ! Merci pour la trad :)